src/components/Header.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function Header() {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+  };
+  return (
+    <header style={headerStyle}>
+      <h1
+        style={{
+          fontSize: '3rem',
+          fontWeight: '600',
+          marginBottom: '2rem',
+          lineHeight: '1em',
+          color: 'skyblue',
+          textAlign: 'center',
```

Comment:
I see here you used inline style for your h1, it's so a bad method you know, but rather than that, I think it would be better to create a dedicated file for styling your content. Kindly try to fix that please